spec: Update language for magic bytes major version (#198)
foxglove/mcap: docs/specification/README.md

    0x89, M, C, A, P, 0x30, \r, \n

> The byte following "MCAP" is the major version byte. `0x30` is the ASCII character `0`. Changes to the spec (i.e. adding fields to records, introducing new records) are binary backward-compatible within the major version.

Reviewer: ```suggestion
The byte following "MCAP" is the major version byte. `0x30` is the ASCII character `0`. Any changes to this specification document (i.e. adding fields to records, introducing new records) will be binary backward-compatible within the major version.
```
